src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
```

Comment:
The class name 'ChatReques6' seems to have a typo. It should be 'ChatRequest' for better readability and adherence to naming conventions.